package.json

```diff
@@ -2,7 +2,7 @@
   "name": "cozy-desktop",
   "version": "0.16.0",
   "description": "File Synchronization Client for Cozy Cloud",
-  "homepage": "https://github.com/cozy-labs/cozy-desktop",
+  "homepage": "https://docs.cozy.io/en/mobile/desktop.html",
```

Comment:
Shouldn't we just disable the cozy-desktop github pages if they are not supposed to be the main entry point for users?